xbmc/epg/EpgContainer.cpp

```diff
@@ -148,7 +149,7 @@ class CEPGContainerStartJob : public CJob
   }
 };
 
-void CEpgContainer::Start(bool bAsync)
+void CEpgContainer::Start(bool bAsync, bool bIsRestart /* = false */)
```

Comment:
This is just a personal opinion but I would make this method private and expose more explicit methods like `StartAsync`, `Restart` etc. Seeing `foo(true, false)` somewhere in a code base is never very clear. Maybe not for this PR though, just a suggestion.